src/styles/emojiFont/index.js

```diff
@@ -0,0 +1,3 @@
+const emojiFont = 'Segoe UI Emoji, Noto Color Emoji';
```

Comment:
Add a comment on the top saying why do we have to use these fonts for web| Desktop?